.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
+
+## Related Issue
```

Comment:
```suggestion
## Related Issue
```
Let's move this to the top so that reviewer can necessary context before reviewing PRs.